commit 1a69709b8e83d53c722f91117c9535d8274fa07d (refs/changes/55/55/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-07-14T00:41:43+00:00 (3 years, 3 months ago)
    
    OS-5508 rsyslog should not send kernel NOTICE to the console

diff --git a/overlay/generic/etc/rsyslog.conf b/overlay/generic/etc/rsyslog.conf
index cc37dfb9..f60451ca 100644
--- a/overlay/generic/etc/rsyslog.conf
+++ b/overlay/generic/etc/rsyslog.conf
@@ -8,7 +8,7 @@ $ModLoad imsolaris
 $ModLoad imtcp
 $ModLoad imudp
 
-*.err;kern.notice;auth.notice			/dev/sysmsg
+*.err;kern.warning;auth.notice			/dev/sysmsg
 *.err;kern.debug;daemon.notice;mail.crit	/var/adm/messages
 
 *.alert;kern.err;daemon.err			operator
